inc/apirest.class.php

```diff
@@ -440,6 +440,7 @@ public function parseIncomingParams($is_inline_doc = false) {
          $parameters['input']->_prefix_filename = [];
 
       } else if (strpos($content_type, "application/x-www-form-urlencoded") !== false) {
+         /** @var array $postvars */
```

Comment:
Do not see the benefit of this one. Sound like this will only make your IDE happy

Reply:
Yes, it is the only benefit. It can make IDE more accurate on code analysis.